### ISSUE TITLE
Some accessibility errors on automated tests

### ISSUE THREAD

**Issue by frankelavsky** [mitvis/website#24]
Hi folks!! I know you're doing such awesome work with your accessible paper-building pipeline, so I thought I could help add a bit of my expertise to the process. (I'm looking at @jonathanzong's [paper from Eurovis](http://vis.csail.mit.edu/pubs/rich-screen-reader-vis-experiences/) in particular here.) Maybe my findings can help make future papers even more awesome?

## Findings
1. Axe Dev Tools found 385 errors, 1 critical, 1 moderate, 335 serious, 1 minor, and 47 that need review. Ideally, you want to make sure you have only minor errors or errors that need review (and you've confirmed they aren't a problem). If I turn on "best practices" the total jumps to 769.
<img width="586" alt="Screenshot. Total Issues 385. Automatic issues 385. Guided issues 0. Needs review 47. Critical 1. Moderate 1. Serious 335. Minor 1. Best practices OFF. WCAG 2.1 AAA." src="https://user-images.githubusercontent.com/16803401/199296092-1aef765b-9186-4a31-ba6a-2ed073e0557f.png">

2. WAVE found 631 errors and 370 contrast errors. You ideally want 0 of either.
<img width="366" alt="Screenshot. 631 Errors. 370 Contrast Errors. 75 Alerts. 7 Features. 1876 Structural Elements. 8239 ARIA." src="https://user-images.githubusercontent.com/16803401/199297456-939c26f3-c916-4175-a930-ed152de6ba50.png">

## Steps to reproduce
1. Install the free chrome extension of [Axe Dev Tools](https://www.deque.com/axe/devtools/)
2. Install the free chrome extension of the [WAVE evaluation tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US)
3. Axe first and note the warnings. Anything Critical, Moderate, or Serious is bad. Minor can usually be left up to the author's discretion.
4. Then run WAVE. You want 0 errors, 0 contrast errors, and at least some ARIA, structure, and features detected.

Make sure not to run WAVE before Axe, since WAVE actually changes the page (and adds content), which can mess with results sometimes.

## Suggested resolution
Ideally, you don't want any automated tests failing. Deque reckons that automated tests can catch about 57% of all errors, but the presence of errors is a strong indicator of more errors to be found in the manual stage of an accessibility evaluation. In my own work in Chartability, the relationship is also pretty strong here, but sometimes work with automated failures is actually not bad once tested manually (since we use SVG + ARIA often in visualization, these can sometimes make automated tests mad but the end experience is okay).

What I'd do is just walk through the problems, most of them are just duplicates of the same few underlying issues, and see if you can improve your process to make sure future papers don't duplicate these issues. It _does_ look like you'll have to reconsider your color scheme though, especially for the use of that red text.

Perhaps at some point we can all have a little meeting together to share findings, tools, and expertise on this front since we both have a similar mission!

**Comment by arvind** [comment]
Wow, this is fantastic!! Thank you for running these battery of tests for us @frankelavsky — I never thought to do that. Lots of useful opportunities here for us to improve our output (and perhaps even roll some of those up into our script!).

**Comment by alundgard** [comment]
Hi all, my apologies, I'm just seeing this Issue. Thanks for sharing all this info @frankelavsky ! FWIW, I believe that vis.mit.edu went through a redesign within the last year, so I'm guessing these style-related errors are relatively new. Making a few slight color adjustments takes care of 99% of errors reported by Deque's Axe DevTools (WCAG 2.1 AA). These adjustments are reflected in the above #35. I'll try to check out the remaining handful of errors as I have time. Thanks again!

**Comment by arvind** [comment]
After #35 and #37, I think these issues have been resolved. Thank you @frankelavsky for flagging them and to @alundgard and @jonathanzong for resolving them!!
